packages/core/src/styles/tokens/tokens.stories.ts

```diff
@@ -242,7 +286,7 @@ export const spacing = () => getTokenTable('globalSpace');
 
 export const typography = () => getTokenTable('globalTypography');
 
-export const color = () => getTokenTable('globalColor');
+export const color = (_args: any, { globals }: any) => getTokenTable('globalColor', globals);
```

Comment:
The args/globals is a storybook api that allows information from the toolbar menu to be passed into stories